### PR TITLE
Removing usages of the deprecated CashFlowCommand

### PR DESCRIPTION
**Pull request by mkit**
Removing usages of the deprecated CashFlowCommand.

### CODE REVIEW

File: client/mock/src/main/kotlin/net/corda/client/mock/EventGenerator.kt [hunk 1]
```diff
@@ -26,16 +28,16 @@ open class EventGenerator(val parties: List<Party>, val currencies: List<Currenc
 
     protected val issueCashGenerator = amountGenerator.combine(partyGenerator, issueRefGenerator, currencyGenerator) { amount, to, issueRef, ccy ->
```
Comment (mnesbit) [inline client/mock/src/main/kotlin/net/corda/client/mock/EventGenerator.kt]: Seems weird that these 3 similar methods have different access levels.

File: finance/src/main/kotlin/net/corda/flows/CashIssueAndPaymentFlow.kt [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+                notary: Party) : this(amount, issueRef, recipient, anonymous, notary, tracker())
+    constructor(request: IssueAndPaymentRequest) : this(request.amount, request.issueRef, request.recipient, request.anonymous, request.notary, tracker())
+
+    @Suspendable
+    override fun call(): Result {
+        subFlow(CashIssueFlow(amount, issueRef, notary))
+        return subFlow(CashPaymentFlow(amount, recipient, anonymous))
+    }
+
+    class IssueAndPaymentRequest(amount: Amount<Currency>,
```
Comment (mnesbit) [inline finance/src/main/kotlin/net/corda/flows/CashIssueAndPaymentFlow.kt]: @CordaSerializable?

File: finance/src/main/kotlin/net/corda/flows/CashPaymentFlow.kt [hunk 3]
```diff
@@ -62,4 +63,6 @@ open class CashPaymentFlow(
         finaliseTx(setOf(recipient), tx, "Unable to notarise spend")
         return Result(tx, anonymousRecipient)
     }
+
+    class PaymentRequest(amount: Amount<Currency>, val recipient: Party, val anonymous: Boolean, val issuerConstraint: Set<Party> = emptySet()) : AbstractRequest(amount)
```
Comment (mnesbit) [inline finance/src/main/kotlin/net/corda/flows/CashPaymentFlow.kt]: @CordaSerializable?

File: finance/src/main/kotlin/net/corda/flows/CashIssueAndPaymentFlow.kt (hunk 2 above)
Comment (mkit) [inline finance/src/main/kotlin/net/corda/flows/CashIssueAndPaymentFlow.kt]: Good spot.

File: finance/src/main/kotlin/net/corda/flows/CashPaymentFlow.kt (hunk 3 above)
Comment (mkit) [inline finance/src/main/kotlin/net/corda/flows/CashPaymentFlow.kt]: Good spot.

**Pull request merged**